scipy/integrate/_odepackmodule.c

```diff
@@ -825,19 +825,26 @@ static struct PyModuleDef moduledef = {
 PyMODINIT_FUNC
 PyInit__odepack(void)
 {
-    PyObject *m, *d, *s;
+    PyObject *module, *mdict;
+
+    module = PyModule_Create(&moduledef);
+    if (module == NULL) {
+        return NULL;
+    }
 
-    m = PyModule_Create(&moduledef);
     import_array();
```

Comment:
Note that `import_array()` is the only call that doesn't need checking, because it's a macro that does the checking itself and returns null already (see `__multiarray_api.h` in numpy).